GrocifyApp.UI/src/components/Products/ProductMeasuresSelector.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
in useApiRequest.makeRequest, change this:

 if (!response.ok) {
     const errorData = await response.json();
     errorMessage = **errorData.error**;
     throw new Error(errorMessage);
 }